functional/keylime_tenant-ima-signature-sanity/test.sh

```diff
@@ -0,0 +1,121 @@
+        rlRun "cat >gpg.script <<EOF
+%echo Generating a basic OpenPGP key
+Key-Type: RSA
+Key-Length: 4096
+Subkey-Type: RSA
+Subkey-Length: 4096
+Name-Real: Joe Tester
+Name-Comment: with no passphrase
+Name-Email: joe@foo.bar
+Expire-Date: 0
+# Do a commit here, so that we can later print 'done' :-)
+%commit
+%echo done
+EOF"    
+        #generate gpg keys which will be genuine
+        rlRun "gpg --batch --pinentry-mode=loopback --passphrase '' --generate-key gpg.script"
+        rlRun "gpg --armor -o gpg-key.pub --export joe@foo.bar"
+        # sign our IMA key file
+        rlRun "gpg --detach-sign -o signature-gpg-genuine.sig ${limeIMAPublicKey}"
+        # sign our allowlist.txt and get get invalid signature for our x509_evm.pem
+        rlRun "gpg --detach-sign -o signature-gpg-fake.sig allowlist.txt"
+    rlPhaseEnd
+
+    rlPhaseStartTest "Verify IMA key using a locally stored GPG signature"
+        rlRun "gpg --verify signature-gpg-genuine.sig ${limeIMAPublicKey}"
+        rlRun "keylime_tenant -v 127.0.0.1 -t 127.0.0.1 -u ${AGENT_ID} --allowlist allowlist.txt --exclude excludelist.txt -f excludelist.txt --sign_verification_key ${limeIMAPublicKey} --signature-verification-key-sig signature-gpg-genuine.sig --signature-verification-key-sig-key gpg-key.pub -c add"
+        rlRun "limeWaitForAgentStatus ${AGENT_ID} 'Get Quote'"
+        rlRun -s "keylime_tenant -c cvlist"
+        rlAssertGrep "{'code': 200, 'status': 'Success', 'results': {'uuids':.*'${AGENT_ID}'" $rlRun_LOG -E
+    rlPhaseEnd
+
+    rlPhaseStartTest "Test that IMA key verification using GPG signature fails for an invalid key"
+        rlRun "gpg --verify signature-gpg-fake.sig allowlist.txt"
+        rlRun -s "keylime_tenant -v 127.0.0.1 -t 127.0.0.1 -u ${AGENT_ID} --allowlist allowlist.txt --exclude excludelist.txt -f excludelist.txt --sign_verification_key ${limeIMAPublicKey} --signature-verification-key-sig signature-gpg-fake.sig --signature-verification-key-sig-key gpg-key.pub -c update" 1
+        rlAssertGrep "WARNING - Unable to verify signature" $rlRun_LOG
+    rlPhaseEnd
+
+    rlPhaseStartTest "Verify IMA key using a downloaded GPG signature and key"
+        rlRun "cp ${limeIMAPublicKey} ."
+        rlRun "keylime_tenant -v 127.0.0.1 -t 127.0.0.1 -u ${AGENT_ID} --allowlist allowlist.txt --exclude excludelist.txt -f excludelist.txt --signature-verification-key-url 'http://localhost:8000/x509_evm.pem' --signature-verification-key-sig-url 'http://localhost:8000/signature-gpg-genuine.sig' --signature-verification-key-sig-url-key gpg-key.pub -c update"
+        rlRun "limeWaitForAgentStatus ${AGENT_ID} 'Get Quote'"
+        rlRun -s "keylime_tenant -c cvlist"
+        rlAssertGrep "{'code': 200, 'status': 'Success', 'results': {'uuids':.*'${AGENT_ID}'" $rlRun_LOG -E
+    rlPhaseEnd
+
+    rlPhaseStartTest "Test that IMA key verification using downloaded GPG signature fails for an invalid key"
+        rlRun "cp ${limeIMAPublicKey} ."
```

Comment:
This has been done already above. What above removing it from a test phase and add it to a setup phase?